src/ripple/ledger/OpenView.h

```diff
@@ -51,9 +54,11 @@ class OpenView
 
     // List of tx, key order
     using txs_map = std::map<key_type,
-        std::pair<std::shared_ptr<
-            Serializer const>, std::shared_ptr<
-                Serializer const>>>;
+        std::pair<std::shared_ptr<Serializer const>,
+        std::shared_ptr<Serializer const>>,
+        std::less<key_type>, qalloc_type<std::pair<key_type const,
+        std::pair<std::shared_ptr<Serializer const>,
+        std::shared_ptr<Serializer const>>>, false>>;
```

Comment:
I even thought about this:
````
using txs_map_mapped_type = std::pair<std::shared_ptr<Serializer const>,
    std::shared_ptr<Serializer const>>;
using txs_map_value_type = std::map<key_type, txs_map_mapped_type>::value_type;
using txs_map = std::map<key_type, txs_map_mapped_type, std::less<key_type>,
                                          alloc_type<txs_map_value_type, false>>;
````
You can go to town with these games all day long... :-)